tsdb/engine/tsm1/file_store.go

```diff
@@ -121,6 +124,13 @@ func (f *FileStore) Count() int {
 	return len(f.files)
 }
 
+// Files returns TSM files currently loaded.
+func (f *FileStore) Files() []TSMFile {
```

Comment:
I added this for testing. I may just remove these accesses so the tests aren't so dependent on the internals.